cmd/limactl/template.go

```diff
@@ -23,6 +24,9 @@ func newTemplateCommand() *cobra.Command {
 		// The template command is still hidden because the subcommands and options are still under development
 		// and subject to change at any time.
 		Hidden: true,
+		PreRun: func(*cobra.Command, []string) {
+			logrus.Warn("`limactl template` is experimental")
```

Comment:
Could you also update https://lima-vm.io/docs/releases/experimental/

Reply:
Documented in f74e152.

Is it time to squash all commits now?

Reply:
> Documented in [f74e152](https://github.com/lima-vm/lima/commit/f74e152799f3112c7e1820b0bf1b8f1a82add529).

Oops, I mean f3c78d44dd73a0883d4ab5f0199f60121d8afa8d (I changed `limactl tmpl` to `limactl template`).